os/tools/compression/src/compress.c

```diff
@@ -162,6 +164,8 @@ static void compress_file(int block_size, int type, char *in_file, char *out_fil
 		ret = LzmaCompress(&out_buf[LZMA_PROPS_SIZE], &writesize, read_buf, (block_size - readsize), out_buf, &propsSize, 0, 1<<13 , -1, -1, -1, -1, 1);
 		if (ret != SZ_OK) {
 			printf("LZMA Compress failed, ret = %d\n", ret);
+			compression_fail = true;
```

Comment:
Only 'compression fail' is special case ?? only this case we should exit?

Reply:
Will add exit to all the failure cases. Thanks.